src/express-middleware.js

```diff
@@ -37,12 +41,21 @@ export function createExpressMiddleware(adapter, middlewareOptions = {}) {
       }
 
       // Lastly, send the response
-      res.send(responseOptions.content || '');
+      const content = responseOptions.content || '';
+      debug('response body: %s', content);
+      res.send(content);
+      res.on('finish', () => {
+        // NOTE: uses undocumented API of http.ServerResponse, but OK for logging code
+        // eslint-disable-next-line no-underscore-dangle
```

Comment:
oh and the justifiication im using is that if logging code outputs something unexpected, its not like the behavior of the module will break